client/lib/plans/constants.js

```diff
@@ -0,0 +1,22 @@
+export const plansList = {
+	'free-plan': {},
+	'premium-plan': {},
+	'business-bundle': {}
+};
+
+export const featuresList = {
+	'google-analytics': {
+		title: 'Google Analytics Integration',
```

Comment:
I'd prefer to limit the data in the constants file to data (slugs, namespaces, bools, etc.), and leave the presentational wording in the components, otherwise we'll run into issues with translating, particularly when switching languages in the translator component.